### DISCUSSION TITLE
How to ensure that GitHub Codespace runs next to PlanetaryComputer Data (westeurope)?

### DISCUSSION THREAD

**Discussion by scottyhq** [microsoft/PlanetaryComputer#354] [Q&A]
Since the Hub is shutting down I was motivated to try these instructions for GitHub Codespaces https://planetarycomputer.microsoft.com/docs/overview/ui-codespaces/. 

But surprisingly, my codespace server keeps popping up in `uksouth` rather than `westeurope` when explicitly selecting `Europe West` for the deployment option:

```
gh codespace create --location WestEurope --display-name mspc
# use uniqueID from above to check location:
gh api /user/codespaces/mspc-pp46j5w46v36p75 --jq .location 
# UkSouth
```

**Comment by TomAugspurger** [comment]
Hmm that's unfortunate. I guess the [Codespcaes docs](https://docs.github.com/en/codespaces/setting-your-user-preferences/setting-your-default-region-for-github-codespaces) don't explicitly say that WestEurope is the Azure West Europe data center. They might have to load balance across regions :/

**Reply by scottyhq** [reply]
Seems like a GitHub Codespaces bug really... so I wrote up the issue here. If others are interested I recommend 'upvoting' the discussion otherwise it might get buried in the barrage of posts :)  https://github.com/orgs/community/discussions/127025
